src/ast/mod.rs

```diff
@@ -1832,6 +1833,11 @@ pub enum Statement {
         name: ObjectName,
         representation: UserDefinedTypeRepresentation,
     },
+
+    Vacuum {
+        options: Vec<(Keyword, Option<String>)>,
```

Comment:
The pattern with the rest of the structs in this crate is to do the parsing into fields (so that downstream crates don't have to re-implement the string matching for different keywords)

So in this case this would mean something like the following

```rust
struct VacuumOptions {
  full: Option<bool>,
  freeze: Option<bool>,
...
}
```